src/modules/bottom-sheet/bottom-sheet.tsx

```diff
@@ -117,24 +123,22 @@ export const BottomSheet: React.FC<IProps> = ({
               <StyledHeader.Title variant="h3">
                 {header.title}
               </StyledHeader.Title>
-
               {header?.description && (
                 <StyledHeader.Description>
                   {header.description}
                 </StyledHeader.Description>
               )}
             </StyledHeader.Wrapper>
           )}
-
           {/* Content */}
           <StyledContentWrapper>{children}</StyledContentWrapper>
-
           {/* Footer */}
           <StyledFooterWrapper>
             <SafeAreaView>{footer}</SafeAreaView>
           </StyledFooterWrapper>
         </StyledSectionWrapper>
       </StyledWrapper>
+      </SafeAreaView>
```

Comment:
used SafeAreaView which works only in ios. This results in margin-top to take effect from the end of the notch for ios devices.